app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/onboarding/address/SubscribedChangeAddressController.scala

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers.address
+package uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers.onboarding.address
```

Comment:
In general, things to do with the account should go under an `account` package - it doesn't make sense to put them under onboarding. If someone has a cgt account, they've already onboarded.

Reply:
fixed